.changeset/nice-zebras-provide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@rspress/shared': minor
+'@rspress/core': minor
+---
+
+support cleanUrls to generate url without suffix
```

Comment:
The changeset info should be like this format:
```
feat: support cleanUrls to generate url without suffix

feat: 支持 `cleanUrls` 参数以去掉 url 的后缀名
```

Reply:
The Chinese message should be optional.

Maybe Rspress do not need a Chinese changelog (or generate by GPT)